src/Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php

```diff
@@ -848,7 +851,9 @@ public function findServiceGroups(): array
             }
         }
 
-        $request ='SELECT SQL_CALC_FOUND_ROWS DISTINCT sg.* FROM `:dbstg`.`servicegroups` sg ' . $subRequest;
+        $request =
+            'SELECT SQL_CALC_FOUND_ROWS DISTINCT sg.* 
+            FROM `:dbstg`.`servicegroups` sg ' . $subRequest;
```

Comment:
are the back quote around the bound dbstg mandatory ?

Reply:
No, but I prefer :)